build/charts/antrea/crds/traceflow.yaml

```diff
@@ -246,6 +246,259 @@ spec:
                   type: object
       subresources:
         status: {}
+    - name: v1beta1
+      served: true
+      storage: true
+      additionalPrinterColumns:
+        - jsonPath: .status.phase
+          description: The phase of the Traceflow.
+          name: Phase
+          type: string
+        - jsonPath: .spec.source.pod
+          description: The name of the source Pod.
+          name: Source-Pod
+          type: string
+          priority: 10
+        - jsonPath: .spec.destination.pod
+          description: The name of the destination Pod.
+          name: Destination-Pod
+          type: string
+          priority: 10
+        - jsonPath: .spec.destination.ip
+          description: The IP address of the destination.
+          name: Destination-IP
+          type: string
+          priority: 10
+        - jsonPath: .spec.liveTraffic
+          description: Trace live traffic.
+          name: Live-Traffic
+          type: boolean
+          priority: 10
+        - jsonPath: .spec.droppedOnly
+          description: Capture only the dropped packet.
+          name: Dropped-Only
+          type: boolean
+          priority: 10
+        - jsonPath: .spec.timeout
+          description: Timeout in seconds.
+          name: Timeout
+          type: integer
```

Comment:
do you want to constrain this to [0, 65535] for consistency with the previous API (which used a `uint16`)? Not very important, but larger timeouts don't really make sense. We could even use a smaller maximum value IMO.

Reply:
yeah, I feel we can use a smaller value, maybe [1, 300]?

Reply:
Changed to [1, 300], let me know if you have concern for this small value.

Reply:
I think this is in the wrong place. This entry is in `additionalPrinterColumns`. The `minimum` and `maximum` constraints are for `schema`.

Reply:
Fixed.